apps/src/maze/MazeVisualizationColumn.jsx

```diff
@@ -14,6 +14,34 @@ const MazeVisualizationColumn = function(props) {
       <GameButtons showFinishButton={props.showFinishButton}>
         <StepButton showStepButton={props.showStepButton} />
         {props.showCollectorGemCounter && <CollectorGemCounter />}
+        {props.showSpeedSlider && (
```

Comment:
should we keep this in here? It's not used for any maze besides neighborhood

Reply:
Ooh I can take that out. Should  I also remove the logic I added to maze.js since neighborhood doesn't use that?

Reply:
I think that makes sense. Maybe add a comment in `neighborhood.js` with the logic we will need for incorporating variable speed so we can quickly implement it there when we are ready to do so?